test/partiql-tests-runner/src/test/kotlin/org/partiql/runner/report/ReportGenerator.kt

```diff
@@ -10,59 +10,84 @@ import java.nio.file.Files
 import kotlin.io.path.Path
 
 class ReportGenerator(
```

Comment:
Think we should stick w/ the prior factoring of `ReportGenerator` similar to what we did for `Report` in the prior review. We should have separate files for the test results of the core and extended tests like what we had talked about.